pkg/osquery/tables/macos_software_update/recommended_updates_table.go

```diff
@@ -0,0 +1,82 @@
+//go:build darwin
+// +build darwin
+
+package macos_software_update
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#cgo darwin LDFLAGS: -framework Cocoa
+#include "sus.h"
+*/
+import "C"
+import (
+	"context"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+var updatesData []map[string]interface{}
+
+type Table struct {
+	logger log.Logger
+}
+
+func RecommendedUpdates(logger log.Logger) *table.Plugin {
+	columns := dataflattentable.Columns()
+
+	tableName := "kolide_macos_recommended_updates"
+
+	t := &Table{
+		logger: log.With(logger, "table", tableName),
+	}
+
+	return table.NewPlugin(tableName, columns, t.generate)
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	var results []map[string]string
+
+	updatesData = nil
+	data := getUpdates()
```

Comment:
I understand it, but this looks weird. Maybe put the `updatesData = nil` down in `getUpdates` with an appropriate comment?

Reply:
Yep, that's a bit clearer, thanks!